File: chapter_2/hero/Ironman.java
package hero;

import fly.FlySky;
import weapon.BimWeapon;

public class Ironman extends Heroes{
	Ironman(){
		weapon = new BimWeapon();
		fly = new FlySky();
	}
	
	public void run() {
		System.out.println("So Fast~");
	}
	public void display() {
		System.out.println("I am Iron man.");
	}
}

File: InClass/[3nd Week] Avengers C++/Ironman.cpp
#include <iostream>
#include "hero.h"

using namespace std;

Ironman::Ironman(){
	weapon = new DeWeapon();
	fly = new NoFly();
}


void Ironman::run(){
    cout << "So Fast~" << endl;
}

void Ironman::display(){
    cout << "I am Iron man." << endl;

}